### DISCUSSION TITLE
Unable to use xpath or css selectors

### DISCUSSION THREAD

**Discussion by ahingorani** [getgauge/taiko#2477] [General]
In the project of gauge-js, I am able to launch browser and search a product using the following:
`step("Search for product <searchItem>", async (item) => {
    await write(item, into(textBox({id:'search_1030'})));
    await press('Enter');
});`

After searching for a product, I wanted to assert whether the product is displayed or not. For this, I used $ selector for xpath as follows:
`step("Verify product is displayed", async function () {
    await click($('//a[@class="product-item-link"][contains(.,"Customized Name Tags for Dogs and Cats")]').exists());
});`

Note: The xpath is working, I verified in the DOM using F12 Developer options tool

When I execute the spec, I am getting following error:
ReferenceError: $ is not defined
at Object.<anonymous>
Can someone please help me understand what is missing? Why am I not able to use xpath or css selectors using $?

Screenshot
![image](https://user-images.githubusercontent.com/87131552/150333424-0887e708-7d65-4449-9f9a-f0cd8976b3cd.png)

**Comment by zabil** [comment]
You need to import `$` from taiko

For example

```
const { click, $ } = require('taiko')
```

**Reply by ahingorani** [reply]
Thank you so much!!!
